js/kucoin.js

```diff
@@ -267,6 +432,8 @@ module.exports = class kucoin extends Exchange {
             let deposit = currency['enableDeposit'];
             let withdraw = currency['enableWithdraw'];
             let active = (deposit && withdraw);
+            let defaultMinAmount = Math.pow (10, -precision['amount']);
```

Comment:
In currencies, `precision` is not an assoc-key-array, it is a scalar. So it does not have `-precision['amount']`, use `-precision` instead ) Basically, this is because a market has two currencies, therefore two precision values, where a currency is just a single currency, therefore a single precision value per currency.